### ISSUE TITLE
panel: question about email sending

### ISSUE THREAD

**Issue by Linux123123** [vilhelmprytz/pterodactyl-installer#137]
**Describe the bug**
So basically we can skip the question about sending emails, because if we set `APP_ENVITONMENT_ONLY=false` we can have the user change it from panel. 

**To Reproduce**
Steps to reproduce the behaviour:
1. Run the script
2. Choose panel
3. Get the question

**Expected behaviour**
It shouldn't ask for it.

**Screenshots**
![image](https://user-images.githubusercontent.com/26341983/103526832-2483c180-4e8a-11eb-83f1-045464852691.png)

**OS (please complete the following information):**
 - OS: All
 - Installation: panel

**Additional context**
It is a lot easier to set them from within the panel. Also most users set the `APP_ENVITONMENT_ONLY=false` anyway, so the script can automatically do it.

**Comment by vilhelmprytz** [comment]
Can we set `APP_ENVIRONMENT_ONLY=false` via `php artisan`?

**Comment by Linux123123** [comment]
No, it doesn't have, but we could use something like `sed` to change it. (just used a bug template...)

**Comment by vilhelmprytz** [comment]
![image](https://user-images.githubusercontent.com/15460993/104641034-8a9def00-56a9-11eb-9ae9-0af3c8041a5c.png)

It's limited to configuring SMTP from the interface, but I agree. I'll fix this.
